raf_native_functions.yaml

```diff
@@ -303,6 +303,7 @@ supported:
   - sigmoid_backward
   - tanh_backward
   - ger
+  - matmul
```

Comment:
does this mean we don't need matmul_backward? I remembered last time you said there was an issue if we don't add it to AutoGrad ?

Reply:
Yes you are correct, if we do not add a custom autograd OP the tracing stops. I was planning on creating separate PRs, so that is why I did not include, but I just pushed the commit.